src/lib/formats/tzx_cas.cpp

```diff
@@ -18,7 +18,7 @@ TZX (currently spectrum only) and spectrum TAP cassette format support by Wilber
         case 0x28:  Select Block
         case 0x2A:  Stop Tape if in 48K Mode
         case 0x2B:  Set signal level
-        case 0x5A:  Merge Block
+		case 0x5A:  Merge Block
```

Comment:
Why the random change in indentation on this line?

Reply:
Maybe in a previous change it stayed like this, but it is now in its place